sklearn/model_selection/_search.py

```diff
@@ -722,15 +860,8 @@ class GridSearchCV(BaseSearchCV):
 
```

Comment:
We could do both. But a doctest helps provide the user a tangible example of expected results.

Some simple example could use a `DummyClassifier` using two strategies, perhaps `most_frequent` and `constant`. This would give us deterministic results that the user can easily verify.